frontend/.eslintrc.json

```diff
@@ -2,7 +2,8 @@
     "env": {
         "browser": true,
         "es2021": true,
-        "jest": true
+        "jest": true,
```

Comment:
why is this needed in the frontend?

Reply:
for some reason, it fails building if I don't have it
I do not know if this is something I introduced with testing but when I removed it, the lining would not work

Reply:
you can keep jest, but jasmine can be removed